pdb2sql/align.py

```diff
@@ -0,0 +1,205 @@
+import numpy as np
+from .pdb2sqlcore import pdb2sql
+from .interface import interface
+from .transform import rot_xyz_around_axis
+
+
+def align(pdb, axis=None, export=True, **kwargs):
```

Comment:
Need to change `axis=None` to `axis=x`.
Using `x` but not `z` to keep consistent with other software such as Haddock and pymol.